### PR TITLE
Feature/improve memoryacc doc

### PR DESCRIPTION
**Pull request by nikita-fuchs**
add examples on importing for NodeJS (people are asking about that all the time) plus a code example.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -6,6 +6,9 @@ plugins:
         on_pre_build: "docs.hooks:pre_build"
 repo_url: 'https://github.com/aeternity/aepp-sdk-js'
 
+theme:
+  name: readthedocs
```
Comment (davidyuk) [inline mkdocs.yml:10]: Please, don't mix the changes from different PRs

File: es/account/memory.js [hunk 2]
```diff
@@ -19,7 +19,11 @@
  * Memory Account module
  * @module @aeternity/aepp-sdk/es/account/memory
  * @export MemoryAccount
- * @example import MemoryAccount from '@aeternity/aepp-sdk/es/account/memory'
+ * @example //JS
+ * import MemoryAccount from '@aeternity/aepp-sdk/es/account/memory'
+ * 
+ * //NodeJS
+ * const { Universal: MemoryAccount } = require('@aeternity/aepp-sdk')
```
Comment (davidyuk) [inline es/account/memory.js:26]:
```suggestion
 * const { MemoryAccount } = require('@aeternity/aepp-sdk')
```
MemoryAccount is not the same as Universal

Comment (davidyuk) [reply es/account/memory.js:26]: This looks inconsistent without other examples that show the same for each module. Shouldn't we update them by adding an example for nodejs? Or maybe instead we can remove all of them and write a general guide of amateurs developers how to import a part of sdk in different environments?

File: es/account/memory.js [hunk 3]
```diff
@@ -59,7 +63,7 @@ function validateKeyPair (keyPair) {
 }
 
 /**
- * In-memory `Account` factory
+ * In-memory `Account` factory. The resulting account `stamp` is used in various functions for setting up and configuring the SDK.
```
Comment (davidyuk) [inline es/account/memory.js:66]: I would say that account just provides and address and a signing method.

File: mkdocs.yml (hunk 1 above)
Comment (nikita-fuchs) [reply mkdocs.yml:10]: ah, that one slipped my attention.

File: es/account/memory.js (hunk 3 above)
Comment (nikita-fuchs) [reply es/account/memory.js:66]: Logging the outcome resulted in something very odd for me, and the address was empty also, can you check please?

File: es/account/memory.js (hunk 2 above)
Comment (nikita-fuchs) [reply es/account/memory.js:26]: This is just a first step for a bigger bounty campaign for contributing to the docs and is supposed to serve as one possible example. Inconsistencies are fine for now, as things will be unified in future work. Having a separate guide for environments is definitely a right thing to do, but for the sake of pragmatism and speed, it should be okay to have that extra line. People that come from google can get going right away without having to browse the documentation for additional setup information. For things that require the SDK to be set up we will have a short note about that.

**Comment by davidyuk** [comment]
> add examples on importing for NodeJS (people are asking about that all the time) plus a code example

can you just send them [this link](https://github.com/aeternity/aepp-sdk-js/blob/develop/docs/guides/import-nodejs.md)?

File: es/account/memory.js (hunk 2 above)
Comment (davidyuk) [reply es/account/memory.js:26]: In my opinion, we don't need these examples, the guide folder already covers different import cases. Do you have ideas on how we can make the guides folder more accessible instead?

File: es/account/memory.js (hunk 3 above)
Comment (davidyuk) [reply es/account/memory.js:66]:
```js
const { Crypto, MemoryAccount } = require('./dist/aepp-sdk');

(async () => {
  const keypair = Crypto.generateKeyPair();
  console.log(keypair);
  const memoryAccount = MemoryAccount({ keypair });
  console.log(memoryAccount);
  console.log(await memoryAccount.address());
  console.log(await memoryAccount.sign('test'));
})();
```
Outputs:
```
$ node test.js
{
  publicKey: 'ak_2aBDpnmDY6k8nTVQQg2QheXn2rZPsjecpwydRqkhzKD9HtKDnB',
  secretKey: '8da5c2aa66f1eff003502f526070a0efcaa66bcd25ef1a3a45535f6b7bb24a8acf06dab6590d534589d61d1357e532cdd48926da366e21ea2d8afdb379776d56'
}
{ isGa: false }
ak_2aBDpnmDY6k8nTVQQg2QheXn2rZPsjecpwydRqkhzKD9HtKDnB
Uint8Array(64) [
  101, 144,  50, 210, 154, 194, 194, 133, 226, 192, 177,
  181, 164, 124, 214, 163, 217,  33, 134, 160,  13,   6,
    0,  58, 169,  80,  97, 121,  23, 198, 254, 195, 118,
   96, 255,  43, 156,  78, 101, 219, 120,  59, 223,   4,
  209, 111,  36, 120, 113,   1, 122,  78, 172, 244, 195,
  107, 208,  23,  72, 144,  11, 245, 131,   7
]
```
it looks ok to me

Comment (nikita-fuchs) [reply es/account/memory.js:66]: ah indeed, I forgot they were async. that gives a good example code code btw, thanks!

**Comment by nikita-fuchs** [comment]
> > add examples on importing for NodeJS (people are asking about that all the time) plus a code example
> 
> can you just send them [this link](https://github.com/aeternity/aepp-sdk-js/blob/develop/docs/guides/import-nodejs.md)?

This is not a thing we're supposed to be doing anymore, things need to be more self-explanatory and require less digging, for a better developer experience requiring less communication.

**Comment by omar-saadoun** [comment]
@davidyuk @mradkov could you please help us with this review to get done? Like I said in other posts, a lot of job is being done to get a proper functional, easy to read and find documentation. You help is very welcome.

**Comment by marc0olo** [comment]
For the moment not relevant. Closing this now.

**Pull request closed without merging**